src/gluonnlp/utils/parallel.py

```diff
@@ -0,0 +1,151 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Utility functions for parallel processing."""
+import threading
+try:
+    import Queue as queue
+except ImportError:
+    import queue
+
+__all__ = ['Parallelizable', 'Parallel']
+
+class Parallelizable(object):
+    """ Base class for parallelizable unit of work, which can be invoked by `Parallel`.
```

Comment:
I believe docstring should not start with whitespace, at least according to all examples in PEP 257 https://www.python.org/dev/peps/pep-0257/